### ISSUE TITLE
Agents require a specific format of the manager endpoint in order to connect 

### ISSUE THREAD

**Issue by plhrja** [damies13/rfswarm#98]
Reporting per request on Discord.

**Describe the bug**
When supplying the manager endpoint (either through ini-file or through runtime flags) __without an ending forward slash__ to an agent, the agent seems to initially connect but subsequently immediately disconnect from the manager (with default logging level)

**To Reproduce**
Steps to reproduce the behavior (locally):
1. Start a rfswrm-manager process with a scenario: `rfswarm-manager -r -n -s scenario.rfs -g 10`
2. Start a rfswarm-agent process __without an ending forward slash for the in the manager endpoint__: `rfswarm-agent -m http://localhost:8138 -g 10` 
3. The following log line indicates the that the error is in the endpoint configuration: `RFSwarmAgent: getjobs(681): [10:8]     Exception: Failed to parse: http://localhost:8138Jobs`

**Expected behavior**
Agent connects and starts running tests.

**Desktop (please complete the following information):**
 - OS: linux-alpine (built from python 3.9 docker image)
 - Browser: Chrome
 - Version: 0.8.1

**Comment by damies13** [comment]
I have managed to reproduce this issue

```
RFSwarmAgent: getscripts(558): [5:5]	 Exception: Failed to parse: http://davesmbpsg:8138Scripts
```

**Comment by damies13** [comment]
Fix for this issue has been merged into branch v0.8.2, will be fixed in this version, or you can pull this branch if you need this fix urgently.
